pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -668,6 +668,16 @@ type PodSandboxConfig struct {
 	//      * localhost/<profile-name>: the profile installed to the node's
 	//        local seccomp profile root
 	//
+	// 3. Sysctls
+	//
+	//      key: security.alpha.kubernetes.io/sysctls
+	//      description: list of safe sysctls which are set for the sandbox.
+	//      value: comma separated list of sysctl_name=value key-value pairs.
```

Comment:
Not all sysctls are permitted. How are we handling the whitelist across shims? Is it up to the runtime to decide which systctls to support? Should this information be surfaced to the kubelet so it can take appropriate action (whatever that may be) for invalid values?